lib/swig/customParamValues.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CUSTOMPARAMVALUES_H
```

Comment:
We're generally **trying** to switch to using ``#pragma once`` rather than old style include guards